L10nSnapshotTests/L10nSnapshotTests.swift

```diff
@@ -181,43 +151,53 @@ class L10nSnapshotTests: L10nBaseSnapshotTests {
         snapshot("14SetHomepage-01", waitForLoadingIndicator: false)
     }
 
+    func test16PasscodeSettings() {
+        navigator.goto(SetPasscodeScreen)
+        snapshot("16SetPasscodeScreen-1-nopasscode")
+        userState.newPasscode = "111111"
+        navigator.performAction(Action.SetPasscodeTypeOnce)
+        snapshot("16SetPasscodeScreen-2-typepasscode")
+
+        userState.newPasscode = "111112"
+        navigator.performAction(Action.SetPasscodeTypeOnce)
+        snapshot("16SetPasscodeScreen-3-passcodesmustmatch")
+
+        userState.newPasscode = "111111"
+        navigator.performAction(Action.SetPasscode)
+        snapshot("16SetPasscodeScreen-3")
+
+        navigator.goto(PasscodeIntervalSettings)
+        snapshot("16PasscodeIntervalScreen-1")
+
+    }
+
     func test17PasswordSnackbar() {
-        let app = XCUIApplication()
-        loadWebPage(url: "http://wopr.norad.org/~sarentz/fxios/testpages/password.html", waitForOtherElementWithAriaLabel: "body")
+        navigator.openURL("http://wopr.norad.org/~sarentz/fxios/testpages/password.html")
         app.webViews.element(boundBy: 0).buttons["submit"].tap()
         snapshot("17PasswordSnackbar-01")
         app.buttons["SaveLoginPrompt.saveLoginButton"].tap()
         // The password is pre-filled with a random value so second this this will cause the update prompt
-        loadWebPage(url: "http://wopr.norad.org/~sarentz/fxios/testpages/password.html", waitForOtherElementWithAriaLabel: "body")
+        navigator.openURL("http://wopr.norad.org/~sarentz/fxios/testpages/password.html")
         app.webViews.element(boundBy: 0).buttons["submit"].tap()
         snapshot("17PasswordSnackbar-02")
     }
 
     func test18TopSitesMenu() {
-        let app = XCUIApplication()
-        let topSites = app.cells["TopSitesCell"]
-        topSites.cells.matching(identifier: "TopSite").element(boundBy: 0).press(forDuration: 2.0)
+        navigator.openURL("http://wopr.norad.org/~sarentz/fxios/testpages/index.html")
```

Comment:
this url (or string) should be a constant?

Reply:
:+1: